src/server/generic_family.cc

```diff
@@ -734,6 +734,43 @@ void GenericFamily::Persist(CmdArgList args, ConnectionContext* cntx) {
     (*cntx)->SendLong(0);
 }
 
+std::optional<int32_t> parseExpireOptions(const CmdArgList args, ConnectionContext* cntx) {
+  int32_t flags = ExpireFlags::EXPIRE_ALWAYS;
```

Comment:
Please start with a captial letter, you can also name it `ParseExpireOptionsOrReply` for clarity